net-misc/stargazer/stargazer-2.408.ebuild

```diff
@@ -1,33 +1,7 @@
-# Copyright 1999-2014 Gentoo Foundation
```

Comment:
Since it looks that the ebuild has been practically rewritten, please revbump it. Possibly keep the old version for a while.